### DISCUSSION TITLE
question about solid's package name

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [solidjs/solid#968] [Q&A] [answered]
Hi. In one of your youtube videos, I noticed that solid team has the new brand name (`@solidjs`). Is it decided/scheduled to use this new brand name in this version or you will wait till next major release?

**Comment by atk** [comment, marked as answer]
I think I saw Ryan write that he wanted to wait until 2.0 to adopt the new name officially.

**Reply by MadadiSaeed** [reply]
Thanks for the response.
